src/cli/commands/snip.ts

```diff
@@ -19,14 +19,35 @@ interface SnipArgs extends MainArgs {
   format?: "rst";
 }
 
-export const doRst = async (
+export const createFormattedCodeBlock = async (
+  result: ProcessResult,
+  destination: string,
+  format: string
+): Promise<void> => {
+  if (format == "rst") {
```

Comment:
Pretty much always use `===`
```suggestion
  if (format === "rst") {
```

Reply:
d'oh!